x-pack/plugin/sql/sql-cli/src/test/java/org/elasticsearch/xpack/sql/cli/CliSessionTests.java

```diff
@@ -26,40 +27,58 @@
 
     public void testProperConnection() throws Exception {
         HttpClient httpClient = mock(HttpClient.class);
-        when(httpClient.serverInfo()).thenReturn(new MainResponse(randomAlphaOfLength(5), org.elasticsearch.Version.CURRENT.toString(),
+        when(httpClient.serverInfo()).thenReturn(new MainResponse(randomAlphaOfLength(5), ClientVersion.CURRENT.toString(),
                 ClusterName.DEFAULT.value(), UUIDs.randomBase64UUID()));
         CliSession cliSession = new CliSession(httpClient);
         cliSession.checkConnection();
         verify(httpClient, times(1)).serverInfo();
         verifyNoMoreInteractions(httpClient);
     }
 
+    public static <T extends Throwable> void expectThrows(Class<T> expectedType, ThrowingRunnable runnable, String exceptionMessage) {
+        T throwable = expectThrows(expectedType, runnable);
+        assertEquals(exceptionMessage, throwable.getMessage());
+    }
+
     public void testConnection() throws Exception {
         HttpClient httpClient = mock(HttpClient.class);
         when(httpClient.serverInfo()).thenThrow(new SQLException("Cannot connect"));
         CliSession cliSession = new CliSession(httpClient);
-        expectThrows(ClientException.class, cliSession::checkConnection);
+        expectThrows(ClientException.class, cliSession::checkConnection, "java.sql.SQLException: Cannot connect");
```

Comment:
That's a good way of wrapping two checks in the same one, but throughout the code in SQL (where there are many such checks) we do them separately: `expectThrows` and then another `assertEquals` on the exception message.

I'm not saying you should change it, but the way it is now, it is inconsistent throughout sql code.

Reply:
adjusted to keep consistency.